.travis.yml

```diff
@@ -1,18 +1,46 @@
 language: python
 
-python:
-- '3.6'
+env:
+  - PYTHON=python
+
+matrix:
+  include:
+    - os: linux
+      python: 3.6
+    - os: linux
+      # Get Python 3.7 from Xenial
+      # https://github.com/travis-ci/travis-ci/issues/9815
+      python: 3.7
+      dist: xenial
+    - os: windows
+      # Get Python 3.7 from Chocolatey
+      language: sh
+      python: 3.7
+      env: PYTHON=py
+      before_install:
+        - choco install python --version 3.7
+        - export PATH="/c/Python37:/c/Python37/Scripts:$PATH"
+        - python -m pip install --upgrade pip wheel
+    - os: osx
+      # Perform the manual steps on osx to install python3
+      language: sh
+      python: 3.7
+      env:
+        - PYTHON=python3
+        - HOMEBREW_NO_INSTALL_CLEANUP=1
+      before_install:
+        # Remove unneeded stuff (which Brew would want to upgrade)
+        - "brew remove numpy gdal postgis postgresql || :"
+        # Install or update Python
+        - "brew install python3 || :"
+        - "brew upgrade python3"
 
 cache:
   directories:
   - $HOME/.cache/pip
 
 install:
-- python -m pip install . 'pytest>=4.0'
+- $PYTHON -m pip install . 'pytest>=4.0'
 
 script:
-- python -m pytest test_naucse_render
-
-after_script:
-- echo "Suplementary logs:"
-- cat .arca/naucse.log
```

Comment:
This is no longer useful?

Reply:
Thanks for catching that.

Reply:
Actually, it is not useful here – `naucse_render` is often called from within Arca, but it (and its tests) doesn't use Arca.